tests/ui/single_call_fn.rs

```diff
@@ -72,3 +72,11 @@ fn k() {}
 fn l() {
     k();
 }
+
+// Make sure lint attributes work properly
+#[allow(clippy::single_call_fn)]
+fn lint_attr_test() {}
+
+fn lint_attr_test_caller() {
+    lint_attr_test();
+}
```

Comment:
What was not working before?

Reply:
Lint attributes, `allow`ing it on the function wouldn't do anything (I think). Similarly with `warn` and others